app/policies/users/company_policy.rb

```diff
@@ -0,0 +1,7 @@
+module Users
+  class CompanyPolicy < ApplicationPolicy
+    def create?
+      user.companies.blank?
```

Comment:
What is the logic behind creating companies?
I see here that a user can't create a company if he already has at least one.
If he's a writer in some company, he still can't create his own company?

Reply:
I need think about it. :)